src/components/Inspector/Files/FileRow.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import CSSModules from 'react-css-modules';
+
+import FileList from './FileList';
+import PriorityButton from './PriorityButton';
+
+import styles from './styles/index.css';
+
+function FileRow({ name, entries }) {
+  return (
+    <div styleName='fileRowContainer'>
+      <div styleName='fileRow'>
+        <input type='checkbox' />
+        <div styleName='name'>{name}</div>
+        <PriorityButton />
```

Comment:
We are missing something here. Those priority buttons should remember your decision, so they should read some property from entries.